src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsMedrecReloading.java

```diff
@@ -26,6 +26,11 @@ public void regularFloatingMedrecIsReloadingWithRecirculationModule() {
     AdsBaseObject ads = new AdsBaseObject(driver, page.getUrl());
 
     ads.scrollToPosition(RECIRCULATION_SELECTOR);
+    try {
+      Thread.sleep(10000);
+    } catch (InterruptedException e) {
+      e.printStackTrace();
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Use a logger to log this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1148)